lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -108,7 +108,7 @@ class PlatformViewManager {
   /// The resulting DOM for the `contents` of a Platform View looks like this:
   ///
   /// ```html
-  /// <flt-platform-view slot="...">
+  /// <flt-platform-view id="flt-platform-view-VIEW_ID" slot="...">
```

Comment:
```suggestion
  /// <flt-platform-view id="flt-pv-VIEW_ID" slot="...">
```

(According to https://github.com/flutter/engine/pull/44188/files#diff-0ee07db46a6272991cd49c1bace8240dc5efaa8a0efabbbfc3f80722589653f1R19)

Reply:
Good catch! I initially named it "platform-view", but then found the slot name and decided to be consistent with that, but forgot to update the docs.